styles/_uswds-theme-custom-styles.scss

```diff
@@ -33,74 +33,3 @@ i.e.
 .main {
   flex: 1;
 }
-
-section {
-  max-width: 96rem;
-  margin-left: auto;
-  margin-right: auto;
-}
-
-// Header:
-// Set custom background and text colors.
-.header {
-  background-color: #162e51;
-  color: #fff;
-}
-
-// Tables:
-// Set the tables to be 100% width.
-.usa-table {
-  width: 100%;
-}
-
-.usa-navbar {
-  max-width: 64rem;
-  margin-left: auto;
-  margin-right: auto;
-}
-
-// Footer logo spacing:
-// Currently the next/image component wraps the <img> tag in a bunch of <spans>
-// and applies direct css onto all those elements. To overcome it, we need
-// to manually override the parent element span using the unfortunate !important
-// See https://github.com/vercel/next.js/discussions/18312
-.footer .logos > img {
-  margin-right: 1.5rem !important;
-}
-
-// USWDS Process List:
-// Remove margin on last child
-.usa-process-list__item:last-child {
-  padding-bottom: 0;
-}
-
-// Custom content group:
-// A logical content grouping with spacing below.
-.content-group,
-.usa-fieldset {
-  @include u-padding-bottom(2);
-
-  p:last-child {
-    margin-bottom: 0;
-  }
-}
-
-// A logical content grouping for content.
-// Used on the confirmation page.
-.content-group-small {
-  @include u-margin-bottom(4);
-
-  h2 {
-    @include u-margin-bottom(0);
-  }
-
-  p {
-    @include u-margin-top(1);
-  }
-}
-
-.text-unstyled {
-  text-decoration-line: none;
-  text-decoration-color: #ffffff;
-  background-color: #005ea2;
-}
```

Comment:
.. this was my friend that made ButtonLinks work in the screener
(not saying we need it, just that we might again need it)

Reply:
Let's take it out of the template, but definitely will pull it into PRP